backend/README.md

```diff
@@ -352,6 +352,12 @@ You can run it this way with Docker:
     docker pull elasticsearch:7.0.0
     docker run -d -p 9200:9200 -p 9300:9300 -v esdata:/usr/share/elasticsearch -v esconfig:/usr/share/elasticsearch/config -e "discovery.type=single-node" -e "cluster.routing.allocation.disk.threshold_enabled=false" elasticsearch:7.0.0
 
+You need also a minio server on port 9000 for s» related test
+You can run it this way with Docker:
+
+    docker pull minio/minio
+    docker run -p 9000:9000 -e MINIO_ACCESS_KEY=minioadmin -e MINIO_SECRET_KEY=minioadmin minio/minio server /data
+
 Create default test_storage_dir folder tree to make test work properly out of the box:
```

Comment:
I think you are missing documentation for installing the s3 storage dependencies (be it for tests or production):
```shell
pip install -r requirements-s3.txt
```

Reply:
Missing also new parameter in docker config file to activate this feature or maybe its just necessary to declare some env variable. Need documentation about that.
Can we install `requirements-s3.txt` by default and just activate S3 if user want to use this ?

Reply:
Yes I think always installing the s3 requirements is ok (it not that big) and easier to handle. The user still has to configure a S3 storage to use it and he can still select a local storage of course.

Reply:
I added the requirements-s3.txt in docker like database. Not sure we should add it to setup_default_backend script (we do only install sqlite driver there)